src/Strategy.js

```diff
@@ -0,0 +1,73 @@
+//@flow
+
+import {expandStyle} from './compiler';
+import {Spec} from './ElementPropSpec';
```

Comment:
Yeap, I will definitely do this, right now I want to monkey patch Spec first. Right now I can't even do that, because style and Element not unified (style is not using Spec in current release)